lib/maths/time_series/CTimeSeriesModel.cc

```diff
@@ -1324,8 +1312,8 @@ std::size_t CUnivariateTimeSeriesModel::memoryUsage() const {
 bool CUnivariateTimeSeriesModel::acceptRestoreTraverser(const common::SModelRestoreParams& params,
                                                         core::CStateRestoreTraverser& traverser) {
     bool stateMissingControllerChecks{false};
-    if (traverser.name() == VERSION_6_3_TAG || traverser.name() == VERSION_7_11_TAG) {
-        stateMissingControllerChecks = (traverser.name() == VERSION_6_3_TAG);
+    if (traverser.name() == VERSION_7_11_TAG) {
+        stateMissingControllerChecks = false;
```

Comment:
Initialised to false.
```suggestion
```

Reply:
However, you can just delete this and the if blocks after the restore loop since they'll never be entered.